### PR TITLE
version 1.1.1

### PR DESCRIPTION
**Pull request by NeonGC**
**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [x] Feature
- [x] Refactor
- [x] Other, please describe:
new version

**Other information:**

Commints:
- remove package-lock.json
- add custom image, refactoring
- optimize styles
- MAJOR (#29): fix WhatsAppButton and fix documentation
- change vkontakte event name option
- code style
- update dependencies
- refactoring
- update build scripts
- update LiveJournal, Odnoklassniki, Pinterest, Vkontakte api, update readme, remove Evernote, fix demo component
- optimize v-bind params
- change version to 1.1.1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .DS_Store
 node_modules
+package-lock.json
```
Comment (Alexandrshy) [inline .gitignore:3]: why are you removing the lock file and ignoring it? I think this is redundant

**Review by Alexandrshy — COMMENTED** [review]
Hey @NeonGC thanks for your amazing work, I just have one question

File: .gitignore (hunk 1 above)
Comment (NeonGC) [reply .gitignore:3]: package-lock.json is used for development, but when publishing a package to npm it is still ignored. Despite the npm documentation, I usually remove it anyway, in order to always have fresh dependencies. If you think it's redundant, then I can return it.

Comment (Alexandrshy) [reply .gitignore:3]: I usually leave the local file, but for projects that have a more complex structure. OK, let's leave your option

**Pull request merged**